examples/Example0_LibrarylessCommunication/Example0_LibrarylessCommunication.ino

```diff
@@ -16,7 +16,12 @@
 // Note that both of these definitions are optional; just prefix either line with // to remove it.
 //  Remove serialNotecard if you wired your Notecard using I2C SDA/SCL pins instead of serial RX/TX
 
-#define serialNotecard Serial1
+#ifndef NOTECARD_SERIAL     
```

Comment:
It's necessary so that we can redefine it in the build. Not all platforms have Serial1. Defines on the command line must be in uppercase.

Reply:
I have never liked `serialNotecard`. I need to see how pervasive it is in the documentation, but I would like to see it replaced with `NOTECARD_SERIAL` everywhere, not just in the precompiler.

Reply:
It's not an API from what I can see, only used in the examples folder, so changing it should not have any further impact. 
If it were an API, we'd probably add a deprecation warning, and phase it out as a breaking change in a later release.  Just for the thought exercise, here's how we might go about doing that:

This will require making `serialNotecard` a reference, something like

```
__attribute__((deprecated)) auto serialNotecard& = NOTECARD_SERIAL;
```